tests/components/api.spec.ts

```diff
@@ -180,12 +191,25 @@ describe('API', () => {
           await api.fileTransfer(Buffer.alloc(0), subscribeMsgs);
           expect(true).to.equals(false);
         } catch (e) {
-          expect(e).to.equals('Error log transfer timed out');
+          expect(e.message).to.equals('Timeout');
           expect(transport.removeAllListeners.callCount).to.equals(1);
           expect(transport.removeAllListeners.getCall(0).args[0]).to.equals('async_file_transfer/timeout');
         }
       });
     });
+
+    describe('on no response', () => {
+      it('should timeout if none of the filetransfer messages are resolved within given time limit', async () => {
+        clock.restore(); // Restore clock
```

Comment:
Better to not use fake clock if it never the less restored all the time.

Reply:
I am using the fake timer to avoid the timeout function that lives now inside fileTransfer.